pkg/proxy/metrics/metrics.go

```diff
@@ -138,6 +140,14 @@ var (
 		},
 	)
 
+	// iptablesCTStateInvalidDroppedPacketsDescription describe the metrics for the number of packets dropped
+	// by iptables which were marked INVALID by conntrack.
+	iptablesCTStateInvalidDroppedPacketsDescription = metrics.NewDesc(
+		"kubeproxy_iptables_ct_state_invalid_dropped_packets",
```

Comment:
Since this metric is a counter, it should be suffixed by "_total" https://prometheus.io/docs/practices/naming/